maintenance_curation.pt.Rmd

```diff
@@ -6,213 +6,215 @@ aliases:
-  dependências reversas ou objetivos estratégicos da rOpenSci.
-
-- Quando arquivados, esses pacotes serão movidos do GitHub "ropensci
-  para a organização "ropensci-archive" (ou autor
-  contas do GitHub do autor, se você desejar), seguindo [orientação de transferência](#archivalguidance). Elas ganharão o tipo "arquivado"
-  no registro. Eles manterão as tags "peer-reviewed" (revisado por pares) e
-  e emblemas. Eles não serão criados no sistema rOpenSci.
-
-- Os pacotes arquivados não serão exibidos por padrão. Uma guia especial de pacotes
-  exibirá esses pacotes com `"type": "archived"`
-  que foram revisados por pares ou mantidos pela equipe.
-
-## Pacotes adquiridos legados {#legacy-acquired-packages}
-
- Os pacotes "legados" são pacotes que não foram criados ou mantidos pela rOpenSci
-e não são revisados por pares, mas estão sob o rOpenSci GitHub
-devido a razões históricas. (Antes de estabelecer a organização
-processo de revisão por pares e seu escopo, a rOpenSci absorveu pacotes de
+  atualização, ou contribuir com algum suporte, ou ainda, buscar um novo mantenedor, ou transferir
+  o pacote para o status "archived".
+
+- Com base no volume de usuários (isto é, o volume de *downloads* do pacote), ou das dependências reversas, ou
+  dos objetivos estratégicos da rOpenSci, a equipe da rOpenSci pode apoiar os
+  pacotes que estiverem com problemas e falhas, por meio de PRs que são revisados pelos autores dos pacotes,
+  ou ainda, com alterações diretas
+  (se os autores não responderem por aproximadamente um mês). A rOpenSci
+  também fornecerá suporte aos autores de pacotes mediante solicitação, tanto pela
+  equipe interna, quanto por voluntários da comunidade, de acordo com o tempo disponível.
+
+- A pedido do autor, ou se os autores não responderem às consultas por aproximadamente um mês,
+  a rOpenSci poderá procurar um novo
+  mantenedor para os pacotes selecionados, que sejam revisados por pares, e que a rOpenSci considere ter alta
+  valor para a comunidade, com base no volume de usuários/*downloads*, ou nas
+  dependências reversas, ou nos objetivos estratégicos da rOpenSci.
+
+- Quando arquivados, esses pacotes serão movidos da organização "ropensci" 
+  para a organização "ropensci-archive" no GitHub (ou para a conta do autor
+  no GitHub, caso for de desejo do autor), seguindo as [orientações de transferência](#archivalguidance).
+  Elas ganharão o tipo "archived"
+  no registro. Esses pacotes vão manter as tags "peer-reviewed" (revisado por pares) e
+  e *badges*. Eles não serão construídos no sistema da rOpenSci.
+
+- Os pacotes arquivados não serão exibidos por padrão na seção de pacotes da
+  página da Web. Esses pacotes serão exibidos em uma guia especial das páginas de pacotes
+  com `"type": "archived"` que foram revisados por pares, ou que foram mantidos pela equipe.
+
+## Pacotes legado que foram adquiridos {#legacy-acquired-packages}
+
+Os pacotes "legados" são pacotes que não foram criados ou mantidos pela rOpenSci
```

Comment:
I the next lines in this file, the translation of "legacy" is not following plural of packages. Shouldn't it follow? If not, then I assume the same should happen with this line:

```suggestion
Os pacotes "legado" são pacotes que não foram criados ou mantidos pela rOpenSci
```